src/kudu/gutil/strings/escaping.h

```diff
@@ -315,16 +315,16 @@ int QEncodingUnescape(const char* src, int slen, char* dest, int szdest);
 //    return false (with dest empty) if src contains invalid chars; for
 //    these versions src and dest must be different strings.
 // ----------------------------------------------------------------------
-int Base64Unescape(const char* src, int slen, char* dest, int szdest);
-bool Base64Unescape(const char* src, int slen, std::string* dest);
+int Base64Unescape(const unsigned char* src, int slen, char* dest, int szdest);
+bool Base64Unescape(const unsigned char* src, int slen, std::string* dest);
 inline bool Base64Unescape(const std::string& src, std::string* dest) {
-  return Base64Unescape(src.data(), src.size(), dest);
+  return Base64Unescape(reinterpret_cast<const unsigned char*>(src.data()), src.size(), dest);
```

Comment:
This was a compiler warning error in fbcode where at some point we're using a `char*` as a `unsigned char*` further down the line. Given that we're already treating it like a `unsigned char*`, we convert it explicitly instead.